src/locale/locales/it/messages.po

```diff
@@ -2964,17 +2964,17 @@ msgstr ""
 
 #: src/view/screens/Settings/index.tsx:539
 msgid "Following feed preferences"
-msgstr "Preferenze del Following feed"
+msgstr "Preferenze del feed Seguiti"
```

Comment:
```suggestion
msgstr "Preferenze del feed Following"
```
The name of the Following feed isn't localized yet unfortunately, so I think it would be clearer to not translate it.